src/app/containers/Headings/helpers.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line no-useless-escape
+const regexPunctuationSymbols = /[\[\]\.,\/#?¿!$'"%^&*;:{}=\-_`~()؟؛٬«»！，。？、@#￥……（）：；《）《》“”〔〕’|]/gi;
```

Comment:
Is there a reason we can't use the pattern we were using before?

https://github.com/bbc/psammead/blob/fba7e7c15fa814d4b7193afeac9e6589894a838b/packages/components/psammead-headings/src/index.jsx#L29-L33

Reply:
it removes anything that's not latin script (ie. persian/arabic/chinese/korean/hindi) as they're not a-z